src/main/java/com/manardenza/entity/User.java

```diff
@@ -0,0 +1,16 @@
+package com.manardenza.entity;
+
+import lombok.Data;
+
+@Data
+public class User extends AbstractObject {
+
+    private String firstName;
+    private String lastName;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "lastName" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)